cmd/tkn/main.go

```diff
@@ -26,7 +29,25 @@ func main() {
 	tp := &cli.TektonParams{}
 	tkn := cmd.Root(tp)
 
-	if err := tkn.Execute(); err != nil {
+	args := os.Args[1:]
+	cmd, _, _ := tkn.Find(args)
+	if cmd != nil && cmd == tkn {
+		pluginCmd := "tkn-" + os.Args[1]
+		exCmd, err := exec.LookPath(pluginCmd)
+		if err != nil {
+			// Can't find the binary in PATH, bailing to usual execution
+			goto CoreTkn
+		}
+
+		if err := syscall.Exec(exCmd, append([]string{exCmd}, os.Args[2:]...), os.Environ()); err != nil {
+			fmt.Printf("Command finished with error: %v", err)
+			os.Exit(254)
```

Comment:
why 254? should it be 127? `Command not found or not in path` ? doesn't `128+<exit-code>` have a different meaning? 🤔

Reply:
Random number :sweat_smile: I'll update to 127 :stuck_out_tongue_closed_eyes: